dist/typings/proteus.d.ts

```diff
@@ -266,7 +266,7 @@ export declare module session {
   }
 
   abstract class PreKeyStore {
-    public prekeys: Array<number>;
+    public prekeys: Array<keys.PreKey>;
```

Comment:
hmm, but this line suggests otherwise, it's an ArrayBuffer:

https://github.com/wireapp/wire-webapp-cryptobox/blob/f6b257751398d2f56bc1ffe4683d76adf75036db/src/main/ts/store/Cache.ts#L38